db/schema.rb

```diff
@@ -181,6 +181,7 @@
     t.integer  "total_copies"
     t.string   "primary_subject"
     t.string   "bnumber",                     :limit => 20
+    t.string   "test_column",                 :limit => 20
```

Comment:
We can get rid of this.

Reply:
It keeps showing up in PRs, and Kevin mentioned we'll keep the field in production, so I think we should keep it in this PR so we don't keep seeing it in commits.

Reply:
That's fine, once this is merged, I'll create a PR to remove it. Thoughts?

Reply:
Kevin mentioned we can keep it since we are standing up a new database in Production.  If Production is going to run the migrations, then `test_column` won't show up.  If Production is going to load the schema, then we do.  If the `test_column` data isn't important, then yes, it's a good ideas to create a PR with a migration to remove it.